.ci/skipped_notebooks.yml

```diff
@@ -504,4 +504,10 @@
     - os:
         - macos-13
         - ubuntu-22.04
-        - windows-2019
\ No newline at end of file
+        - windows-2019
+- notebook: notebooks/keras-with-openvino-backend/keras-with-openvino-backend.ipynb
+  skips:
+    - os:
+        - windows-2019
+    - python:
+        - '3.12'
```

Comment:
is it related to your comment about tensorflow-text? I think it will be more informative if you add recommendation to downgrade python to 3.11 on windows if tokenizer does not work

Reply:
No. tensorflow-text doesn't provide package for macOS x86-64 since 2.16.1 version. And this version doesn't have package for python 3.12 and for macOS ARM.